src/components/Item/SearchBar.js

```diff
@@ -60,6 +79,8 @@ export default function CustomizedInputBase() {
   );
 }
 
+export default connect(null, { loadPostingsByQuery })(CustomizedInputBase);
+
 // import React, { Component } from "react";
```

Comment:
Do we still need the code commented out from lines 84-189? Delete-able?

Reply:
This was leftover from Peng. Didn't know if he still wanted it or if we wanted to use his original

Reply:
Gotcha. @Cainuse, still need these? :O

Reply:
btw @jenessatan , I forgot to mention that for this commit, since we still don't have confirmation from Peng, we can just ignore the commented out code for now :D

Reply:
I think you can do whatever. Commented out code doesnt mean anything. I dont think peng is working on FE at all so you do what makes the most sense.